build/dependencies.props

```
@@ -3,7 +3,7 @@
     <BenchmarkDotNetPackageVersion>0.12.1</BenchmarkDotNetPackageVersion>
     <GoogleProtobufPackageVersion>3.12.2</GoogleProtobufPackageVersion>
     <GrpcDotNetPackageVersion>2.30.0</GrpcDotNetPackageVersion>  <!-- Used by example projects -->
-    <GrpcPackageVersion>2.30.0</GrpcPackageVersion>
+    <GrpcPackageVersion>2.31.0-pre2</GrpcPackageVersion>
```

Why pre2? We don't have a pre1.
```suggestion
    <GrpcPackageVersion>2.31.0-pre1</GrpcPackageVersion>
```

---

Grpc.Core pre2 is the latest what we have currently.  (note that this is not version of grpc-dotnet, it's the version of Grpc.Core we import).

---

Ok. I guess it makes sense to keep them in sync. grpc-dotnet pre1 depending on Grpc.Core.Api pre2 would be weird.